etc/smashbox.conf.template

```diff
@@ -136,3 +136,10 @@ share_sets = 1
 #
 # scp port to be used in scp commands, used primarily when copying over the server log file
 scp_port=22
+
+#
+# properties for defining directory depth and directory layout
+#
+dir_depth = 5
```

Comment:
As said on the other PR, these are not required, as they are filled in/overwritten by the [test suite](https://github.com/owncloud/smashbox/pull/76/files#diff-f581357b03ae83b2f39a524395f892eeR41) anyway